packages/storybook-helpers/main.ts

```diff
@@ -37,7 +37,9 @@ const staticDocumentsRule = {
 const config: StorybookConfig = {
   stories: [
     "../src/**/*.stories.@(js|jsx|ts|tsx|mdx)",
+    "../src/**/*.mdx",
     "../../../packages/**/src/**/*.stories.@(js|jsx|ts|tsx|mdx)",
+    "../../../packages/**/src/**/*.mdx",
```

Comment:
Starting in Storybook 7 the `stories.mdx` extension is deprecated.  I'm not sure how the Dialog one is rendering still.

Reply:
Deprecated but **not yet** removed.

REF: https://storybook.js.org/docs/react/writing-docs/mdx

> If you’re coming from a previous version of Storybook, you might be accustomed to using MDX both for documentation and for defining stories in the same .stories.mdx file. We’ve deprecated this functionality and plan to remove it in a future version of Storybook. We provide [migration](https://storybook.js.org/docs/react/writing-docs/mdx#automigration) scripts to help you onto the new format.